server/systemd/novaboot-unfsd-start

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+NB_USER=$1
+
+mkdir -p "$HOME/.unfsd"
+
+PORT_FILE="$HOME/.unfsd/$NB_USER.ports"
+EXPORTS_FILE="$HOME/.unfsd/$NB_USER.exports"
+
+NFS_EXPORTED_PATH="$HOME/tftproot/$NB_USER"
+
+# get network from config file
+. "${NOVABOOT_SHELL_CONFIG:-$HOME/.novaboot-shell}"
+NETWORK=${nfs_network:?}
```

Comment:
If this script is implemented in the `.service`, this would have to be configured differently. Perhaps via `Environment=` in the `.service` file. User will be able to change the value via `systemctl --user edit`.

Or it could be configured via `EnvironmentFile=` and the same file would be used as `ConditionPathExists=`. This would be nice because no service would be started automatically before the administrator configures it.